doc/api/modules.md

```diff
@@ -768,6 +768,15 @@ added: v0.1.16
 
 The module that first required this one.
 
+### module.paths
+<!-- YAML
+added: v0.4.0
+-->
+
+* {string[]}
+
+The search paths for the module. 
```

Comment:
I agree that a few more words should be added. May also a link to the parts about module resolution.
@atever would you like to submit another PR?